examples/s2i-nginx/test/index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+		<meta charset="utf-8">
+		<title>openshift/source-to-image tutorial</title>
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<!-- ... -->
+		<meta name="description" content="source-to-image tutorial example page">
+		<meta name="author" content="Palade Victor">
+	</head>
+	<body>
+  <xmp theme="readable" style="display:none;">
+Creating a basic S2I builder image
```

Comment:
now you've got this content in multiple places.  just make this page do a basic hello world instead.